edward/inferences.py

```diff
@@ -191,12 +276,18 @@ def build_loss(self):
                 return self.build_reparam_loss()
 
     def build_score_loss(self):
-        """
-        Loss function to minimize, whose gradient is a stochastic
-        gradient based on the score function estimator.
-        (Paisley et al., 2012)
+        """Loss function to minimize. 
+
+        Defines a stochastic gradient of
```

Comment:
Defines a loss function whose automatic differentiation is the stochastic gradient of